rebar.lock

```diff
@@ -29,7 +29,10 @@
   {git,"https://github.com/aeternity/enacl.git",
       {ref,"26180f42c0b3a450905d2efd8bc7fd5fd9cece75"}},
   0},
- {<<"enoise">>,{pkg,<<"enoise">>,<<"1.0.1">>},0},
+ {<<"enoise">>,
+  {git,"https://github.com/aeternity/enoise.git",
+      {ref,"1e6ee6703fc401a46fce8f1a400a68734c0a356d"}},
```

Comment:
https://github.com/aeternity/enoise/compare/v1.0.1...1e6ee6703fc401a46fce8f1a400a68734c0a356d

Reply:
I see v1.0.1 is c06b that is included in https://github.com/aeternity/enoise/commits/1e6ee6703fc401a46fce8f1a400a68734c0a356d

Reply:
This is mostly stricter type specs. There are two improvements cc @hanssv  :
* `encrypt_with_ad` used to return misleading ok tuple with nested error tuple rather than binary. This was going to fail in the following code that assumed binary. Ok.
* [`enoise_crypto:decrypt`](https://github.com/aeternity/enoise/commit/7c7ad54a6ac42854a9ee47c86dbbcd035e3b19c8#diff-2f5310cdb96654729bb4e4f5b13beaadR84) used to returned malformed error tuple. This was going to fail in the following code that assumed tuple. Ok.